pgmpy/sampling/Sampling.py

```diff
@@ -156,10 +197,13 @@ def likelihood_weighted_sample(self, evidence=None, size=1):
             None if no evidence
         size: int
             size of sample to be generated
+        return_type: string
```

Comment:
@khalibartan Add the options for the string in the first line. Something like `return_type: string (dataframe | recarray)`.